tests/integration/helpers/givenpage.py

```diff
@@ -43,12 +43,16 @@ class LoadedHtmlPage(object):
     def __init__(self, driver):
         self._driver = driver
 
+    @staticmethod
+    def _set_to_ms(timeout):
+        return timeout * 1000
```

Comment:
There are two `_set_to_ms` methods in the PR. I think it should be one function somewhere in the root helper of `/tests`.